packages/pyyaml/meta.yaml

```
@@ -0,0 +1,28 @@
+package:
+  name: pyyaml
+  version: 5.3.1
+
+source:
+  # path: downloads/pyyaml-5.3.1
+  url: https://github.com/yaml/pyyaml/archive/5.3.1.zip
+
+  patches:
+    - patches/Makefile.patch
+    - patches/setup.cfg.patch
+
+build:
+  skip_host: True
+  cflags: |
+    -I/src/cpython/installs/python-3.8.2/include/python3.8
+    -I/src/emsdk/emsdk/fastcomp/emscripten/system/include/libc
+    -I/src/packages/libyaml/build/include
+  ldflags: |
+    -L/src/packages/libyaml/build/lib
+
+requirements:
+  run:
+    - libyaml
+
+test:
+  imports:
+    - pyyaml
```

The issue is that this should be,
```suggestion
    - yaml
```
I think. As it's also used for determining import > package mapping.

---

It's a bit confusing to me that the field which determines import > package mapping is under `test`. I guess the docs say `test/imports` is the "List of imports to test after the package is built." which is taken also the set of python imports to expose that depend on this package.

Presumably having `pyyaml` here would have caused the packages tests to fail on `ImportError` when attempting to import `pyyaml` if the packages were built successfully.

---

Yes, is currently a bit confusing but we are also trying to follow conda's [meta.yaml spec](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html) as much as possible. Not sure that changing the name of the top level key would be worth it. Maybe we just need better documentation.